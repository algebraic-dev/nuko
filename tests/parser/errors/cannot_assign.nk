let a =
    let a = 3