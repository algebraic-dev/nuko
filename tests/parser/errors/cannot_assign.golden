• These
AssignInEndOfBlock
    ( Range
        { start = Pos
            { line = 1
            , column = 4
            }
        , end = Pos
            { line = 1
            , column = 13
            }
        }
    )

 └Program
   ├typeDecls: Empty List
   ├letDecls:
   ┆ └List
   ┆   └LetDecl
   ┆     ├declName: (Name "a" 0:4-0:5)
   ┆     ├declArgs: Empty List
   ┆     ├declBody:
   ┆     ┆ └Lit
   ┆     ┆   ├LInt 1:12-1:13 3
   ┆     ┆   └∅
   ┆     ├declRet: Nothing
   ┆     └declExt: ∅
   ├impDecls: Empty List
   └programExt: ∅
